private/buf/bufprint/bufprint.go

```diff
@@ -73,66 +78,182 @@ func (f Format) String() string {
 	}
 }
 
-// CuratedPluginPrinter is a printer for curated plugins.
-type CuratedPluginPrinter interface {
-	PrintCuratedPlugin(ctx context.Context, format Format, plugin *registryv1alpha1.CuratedPlugin) error
-	PrintCuratedPlugins(ctx context.Context, format Format, nextPageToken string, plugins ...*registryv1alpha1.CuratedPlugin) error
+// OutputEntity is an entity printed structurally by functions in bufprint package.
+// It's used in "buf registry" commands where the CLI prints a page of entities, such as
+// commits, an entity's info or simply an entity's full name.
+//
+// An implementation of OutputEntity must also be a struct. If a field should be printed
+// in table form, add a field tag with bufprint:"<field name>[,omitempty]".
+type OutputEntity interface {
```

Comment:
Entity

Reply:
fixed in [c5b6335](https://github.com/bufbuild/buf/pull/3173/commits/c5b6335f472fee63a68d1c10ac32c12d1ee76b77)